deepchem/models/tensorgraph/models/robust_multitask.py

```diff
@@ -0,0 +1,89 @@
+import tensorflow as tf
+from deepchem.models.tensorgraph.tensor_graph import MultiTaskTensorGraph
+from deepchem.models.tensorgraph.layers import Input, Dense, Concat, SoftMax, SoftMaxCrossEntropy, Layer
+
+
+class WeightedError(Layer):
+
+  def __call__(self, *parents):
+    entropy, weights = parents[0], parents[1]
+    self.out_tensor = tf.reduce_sum(entropy.out_tensor * weights.out_tensor)
+    return self.out_tensor
+
+
+def tensorGraphMultitaskClassifier(n_tasks,
+                                   n_features,
+                                   layer_sizes=[500],
+                                   bypass_layer_sizes=[100],
+                                   model_dir=None):
+  """
+  TODO(LESWING) Add Dropout and regularization
+  
+  Parameters
+  ----------
+  n_tasks
+  n_features
+  layer_sizes
+  bypass_layer_sizes
+  model_dir
+
+  Returns
+  -------
+
+  """
+  g = MultiTaskTensorGraph(model_dir=model_dir)
+  in_layer = Input(shape=(None, n_features), name="FEATURE")
+  g.add_layer(in_layer)
+  g.add_feature(in_layer)
```

Comment:
I added it to add_label, but I don't want to add parents=[] as a param to set_loss which is required because a loss has to be wired up.